internal/aez/ccd_power_regulators.go

```diff
@@ -25,6 +25,7 @@ type CPRU struct {
 	VOD3   uint16 // CCD3 Output Drain Voltage 0..4095
 }
 
-func (s *CPRU) read(buf io.Reader) error {
-	return binary.Read(buf, binary.BigEndian, s)
+// Read CRPU
+func (cpru *CPRU) Read(buf io.Reader) error {
```

Comment:
how about a test of a real CPRU buffer from a `.rac` file resulting in the expected CPRU struct?